descriptor/pom.xml

```diff
@@ -26,20 +26,34 @@
         <plugins>
             <plugin>
                 <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-platform-descriptor-json-plugin</artifactId>
-                <version>${quarkus.version}</version>
+                <artifactId>quarkus-platform-bom-maven-plugin</artifactId>
+                <version>0.0.10</version>
                 <executions>
                     <execution>
                         <phase>process-resources</phase>
                         <goals>
-                            <goal>generate-platform-descriptor-json</goal>
+                            <goal>generate-platform-descriptor</goal>
                         </goals>
                     </execution>
                 </executions>
                 <configuration>
                     <bomArtifactId>quarkus-universe-bom</bomArtifactId>
                     <overridesFile>${overridesfile}</overridesFile>
                     <skipCategoryCheck>true</skipCategoryCheck>
+                    <processGroupIds>
+                        <groupId>org.kie.kogito</groupId>
+                        <groupId>org.optaplanner</groupId>
+                        <groupId>com.blazebit</groupId>
+                        <groupId>io.quarkiverse.freemarker</groupId>
+                        <groupId>io.quarkiverse.googlecloudservices</groupId>
+                        <groupId>io.quarkiverse.minio</groupId>
+                        <groupId>io.quarkus</groupId>
+                        <groupId>org.apache.camel.quarkus</groupId>
+                        <groupId>io.debezium</groupId>
+                        <groupId>org.amqphub.quarkus</groupId>
+                        <groupId>com.datastax.oss.quarkus</groupId>
+                        <groupId>com.hazelcast</groupId>
```

Comment:
@gsmet we should be careful about this list when adding new members. We should probably introduce support for wildcards, e.g. `io.quarkiverse.*`, since anything coming from that group will include extensions.

Reply:
Yes support for wildcards would be nice.